### PR TITLE
Test Math.hypot argument coercion

### PR DESCRIPTION
**Pull request by gibson042**
https://bugs.webkit.org/show_bug.cgi?id=229843

### CODE REVIEW

File: test/built-ins/Math/hypot/Math.hypot_Zero_2.js [hunk 1]
```diff
@@ -2,12 +2,16 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+esid: sec-math.hypot
```
Comment (jugglinmike) [inline test/built-ins/Math/hypot/Math.hypot_Zero_2.js:5]: Thanks for adding this!

**Review by jugglinmike — APPROVED** [review]
Nice catch! I'm curious: how did you discover the JSC bug?

**Comment by gibson042** [comment]
> Nice catch! I'm curious: how did you discover the JSC bug?

I was reviewing the definition of Math.hypot as part of the https://github.com/js-choi/proposal-bigint-math discussion and was struck by the divergent handling of infinities vs. NaN, which I explored with eshost.

**Comment by jugglinmike** [comment]
It's like spider-sense for test bugs :)

**Pull request merged**